gridstatus/isone_api/isone_api.py

```diff
@@ -613,15 +613,15 @@ def _handle_load_forecast(
             "Interval End": "Interval End",
             "CreationDate": "Publish Time",
             "ReliabilityRegion": "Location",
-            "LoadMw": "Load",
+            "LoadMw": "Load Forecast",
             "ReliabilityRegionLoadPercentage": "Regional Percentage",
         }
         system_cols = {
             "BeginDate": "Interval Start",
             "Interval End": "Interval End",
             "CreationDate": "Publish Time",
-            "LoadMw": "Load",
-            "NetLoadMw": "Net Load",
+            "LoadMw": "Load Forecast",
+            "NetLoadMw": "Net Load Forecast",
```

Comment:
Do we need to update the corresponding tests?

Reply:
yar